fe/fe-core/src/main/java/com/starrocks/lake/LakeTableHelper.java

```diff
@@ -206,6 +206,52 @@ public static boolean isSharedDirectory(String path, long physicalPartitionId) {
         return !path.endsWith(String.format("/%d", physicalPartitionId));
     }
 
+    /**
+     * For tables created in the old version of StarRocks cluster, such as v3.2, the column unique id in FE might
+     * have default value -1. When upgrading from old version to v3.3 upwards, the unique id of columns might introduce
+     * compatibility issues. This method is used to check if the table needs to restore the column unique id.
+     */
+    public static void restoreColumnUniqueIdIfNeeded(OlapTable table) {
+        boolean needRestore = false;
+        for (MaterializedIndexMeta indexMeta : table.getIndexIdToMeta().values()) {
+            List<Column> columns = indexMeta.getSchema();
```

Comment:
Just call `restoreColumnUniqueIdIfNeeded(columns)` directly? Is there necessary to call  `LakeTableHelper.restoreColumnUniqueId(table);`?